rules/do-you-create-a-report-whenever-you-need-a-number-from-a-system/rule.md

```diff
@@ -49,4 +49,6 @@ Figure - Bad Example: If you can't build it yourself, send an email to your boss
 ::: good
 Figure - Good Example: Sate your own curiosity with Power BI  
 :::
- See Ulysses' video on how Power BI is the best tool for this: https://tv.ssw.com/6839/power-bi-finally-can-make-decisions-based-facts-ulysses-maclaren
+ See Ulysses' video on how Power BI is the best tool for this: 
+ 
+**Video** - [Power BI - Decisions based on facts: A Real World Look at Power BI](https://www.youtube.com/watch?v=Y4tvF0DWEec)
```

Comment:
Hey @Hona 

The way it was done you only have a link
See how to embed a video: https://ssw.com.au/rules/rule/#8-videos